### DISCUSSION TITLE
Scenic-ArchR

### DISCUSSION THREAD

**Discussion by dimurali93** [aertslab/scenicplus#85] [Q&A]
Have anyone ever implemented scenic for Archr multiome?

**Comment by SeppeDeWinter** [comment]
Hi @dimurali93 

There is no specific implementation for ArchR, however if you are able to extract count matrices and cistromes in the correct format and load it in python you can use the output.

See https://scenicplus.readthedocs.io/en/latest/faqs.html for some ideas.

Let me know if you get stuck somewhere.

Best,

Seppe

**Reply by LacquerHed** [reply]
Also trying to do this. I can export a dgcMatrix from an ArchR RangedSummarizedExperiment however this does not appear to be the proper format for creating a CistopicObject. Any tips on how to do this? Thanks.

**Reply by Umaarasu** [reply]
I am not sure about ArchR, but I recently exported seurat multiome for cisTopic tutorial mentioned in the FAQs to be used for scenic+ analysis. We need to input the count matrix as dataframe with cells as columns and regions as rows and can be obtained with this command "count_matrix <- as.data.frame(seuratObj@assays$ATAC@counts)" and metadata can be done obtained this command  "metadata <- as.data.frame(seuratObj@active.ident)"

**Reply by bioarpit1** [reply]
hi Seppe 

Thanks for the cool tool. The tutorial link you shared above is broken; I tried from  here: https://pycistopic.readthedocs.io/en/latest/faqs.html as well, but that specific tutorial you mentioned has a broken link. Can you please direct me to the correct location? 

Thanks

Arpit

**Reply by SeppeDeWinter** [reply]
Hi @bioarpit1 

You can still find the tutorial here: https://github.com/aertslab/pycisTopic/blob/old/notebooks/Toy_melanoma-RTD.ipynb

Best,

Seppe
